src/tools/pkcs15-tool.c

```diff
@@ -1645,6 +1658,21 @@ static int list_apps(FILE *fout)
 	return 0;
 }
 
+
+static void print_supported_algo_info_operations(unsigned int operation)
+
+{
+	size_t i;
+	const char *operations[] = {
+		"compute_checksum", "compute_signature", "verify_checksum", "verify_signature",
+		"encipher", "decipher", "hash", "generate/derive_key"
+	};
```

Comment:
clang-format suggested the following change:


```
-		"encipher", "decipher", "hash", "generate/derive_key"
-	};
+		"encipher", "decipher", "hash", "generate/derive_key"};
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->